manifests/params.pp

```diff
@@ -109,7 +109,11 @@
       $sysconfig            = '/etc/default/snmpd'
       $var_net_snmp         = '/var/lib/snmp'
       $varnetsnmp_perms     = '0755'
-      $varnetsnmp_owner     = 'snmp'
+      if $lsbdistcodename == "bionic" {
```

Comment:
Instead of this, I think you'd want something like `if versioncmp($::lsbdistrelease, '16.04') > 0 {` so that in  future releases you don't have to come back here and update this again to include whatever release name that has.

Reply:
Makes sense to me, updated